resource_multi_week_work_time_from_contracts/views/hr_contract_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+SPDX-FileCopyrightText: 2024 Coop IT Easy SC
+
+SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<odoo>
+    <record model="ir.ui.view" id="hr_contract_view_form">
+        <field name="name">hr.contract.form</field>
+        <field name="model">hr.contract</field>
+        <field name="inherit_id" ref="hr_contract.hr_contract_view_form" />
+        <field name="arch" type="xml">
+            <field name="resource_calendar_id" position="attributes">
+                <attribute
+                    name="domain"
+                >[('parent_calendar_id', '=', False)]</attribute>
```

Comment:
shouldn’t it better to add this domain on the field in the model directly, so that it applies everywhere?